app/models/dashboard_alert.rb

```diff
@@ -33,6 +33,7 @@ class DashboardAlert < ApplicationRecord
   belongs_to :find_out_more, optional: true
   belongs_to :content_version, class_name: 'AlertTypeRatingContentVersion', foreign_key: :alert_type_rating_content_version_id
 
+  attribute :dashboard, :string
```

Comment:
was this to fix a spec failure somewhere?

Reply:
removed this one, only in app/models/staff_role.rb as rails complaining about non database back enum